spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -594,3 +596,8 @@ class SpdxExpressionTest : WordSpec() {
         }
     }
 }
+
+private infix fun <T> T.shouldBeString(expected: String) {
+    assertionCounter.inc()
+    eq(this.toString(), expected)?.let(errorCollector::collectOrThrow)
```

Comment:
I'm having a hard time understanding what this does. I guess you copied this from somewhere? Can you link to the source or add some code comments why some counter (?!) needs to be incremented here?

Reply:
I decided to implement this with a custom matcher instead. It's a bit more code but I find it cleaner because it does not have to mess with kotest internals.